src/libraries/System.Text.Json/src/System/Text/Json/Reader/JsonReaderHelper.netstandard.cs

```diff
@@ -4,18 +4,32 @@
 using System.Diagnostics;
 using System.Numerics;
 using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
 
 namespace System.Text.Json
 {
     internal static partial class JsonReaderHelper
     {
-        private static unsafe int IndexOfOrLessThan(ref byte searchSpace, byte value0, byte value1, byte lessThan, int length)
+        /// <summary>IndexOfAny('"', '\', less than 32)</summary>
+        /// <remarks>https://tools.ietf.org/html/rfc8259</remarks>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
```suggestion
```

I don't think we'd want to inline the actual implementation

Reply:
Nevermind, I didn't add it, but by combining the other function into it I did end up negatively impacting this. I'll remove it.